addons/website_sale/views/templates.xml

```diff
@@ -818,9 +820,7 @@
                                 t-att-value="c.id"
                                 t-att-checked="'true' if c.id == category.id else None"/>
                             <div
```

Comment:
_Note for the future_: We should remove this `<div>` and move btn classes to the main `<a>` tag.
This would simplify handling focus states and keyboard navigation.